src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -836,6 +836,10 @@ public enum NavLinkMatch
 }
 namespace Microsoft.AspNetCore.Components.Services
 {
+    public partial interface IComponentContext
```

Comment:
How did you update this? Can you try and get the private members of the components that we ship to show up? That breaks the current E2E test in ProjectTemplates. It would be incredibly dope if we updated and enable that.

Reply:
I ran `GenerateReferenceSource` as per https://github.com/aspnet/AspNetCore/blob/master/docs/ReferenceAssemblies.md

What you see here is what it outputs.

Reply:
I do believe we did this for some other APIs in the assembly. This is an ongoing problem on the templates. Things like `Assembly="typeof(Blah).Assembly"` won't work because it won't be able to find the Assembly property on the component and it will treat it as a string instead when compiling.

Reply:
Yes, @rynowak fixed that by telling the tool not to generate anything for those types, and he wrote those ref types manually. I don't think we need to do `IComponentContext` manually though.

Reply:
For now we need to manually generate the reference source of components because non-public methods do not show up in reference assemblies. This should not apply to anything else we need to do. If the contagion starts spreading I would love to learn more about that 😆